wordpress-mobile/WordPress-iOS
# [Gutenberg] Image preview for thumbnail image sizes

This PR implements changes from https://github.com/wordpress-mobile/gutenberg-mobile/pull/1700 to fix an issue previewing images from the image block set to thumb sizes.

`Gutenberg-mobile` side PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1700

This PR also modifies the cache key, to be able to get the cached image from outside the ImageLoader class.

![small_preview](https://user-images.githubusercontent.com/9772967/71173968-536dcc00-2264-11ea-9366-9f8200030f20.gif)

To test:
- Open Gutenberg on WPiOS, on a self-hosted site.
- Add a new image.
- Change its size to thumb.
- Tap on the image to show the image preview.
- Check that the image shown is the original full size image.
- Check that the image object is properly obtained from cache.

PR submission checklist:
